src/pygeoprocessing/routing/routing.pyx

```diff
@@ -2948,6 +2983,114 @@ def extract_streams_mfd(
     LOGGER.info('100.0% complete')
 
 
+def detect_outlets(d8_flow_dir_raster_path_band, target_outlet_vector_path):

```

Comment:
I think this somehow made its way into this PR, though it's also under review in #148 ... I think that wasn't the intent?

Reply:
Sure wasn't the intent and honestly not sure how it got in there. And double honestly I'm still not comfortable with git's branching, I can't seem to unroll it/detect what happened as easily as mercurial. Anyway, I've removed `detect_outlets` in 593996ac11670eb709a1327fe98bf961e4369f7f.